### PR TITLE
snap: when running `snap repair` without arguments, show hint

### PR DESCRIPTION
**Pull request by mvo5**
When running `snap repair` without argument we show nothing right
now. This is not very useful. This PR changes the code so that
we show a hint like:
```
$ ./snap repair
error: no `<repair-id>` given, did you mean 'snap repairs' to list all repairs?
```

### CODE REVIEW

File: cmd/snap/cmd_repair_repairs.go [hunk 1]
```diff
@@ -69,6 +69,10 @@ func init() {
 }
 
 func (x *cmdShowRepair) Execute(args []string) error {
+	if len(x.Positional.Repair) == 0 {
+		return fmt.Errorf("no `<repair-id>` given, did you mean 'snap repairs' to list all repairs?")
```
Comment (pedronis) [inline cmd/snap/cmd_repair_repairs.go]: why the \`\`  ?   also they can be many so perhaps `no <repair-id>s given`  ?

Comment (degville) [reply cmd/snap/cmd_repair_repairs.go]: fwiw, I'm not too keen on questions being asked in command output - it's like we're admitting there's an ambiguity in the design. Maybe:

`error: no <repair-id> given. Try 'snap repairs' to list all repairs or specify a specific repair id.`

Comment (mvo5) [reply cmd/snap/cmd_repair_repairs.go]: I removed the '`' now and added the 's'

Comment (mvo5) [reply cmd/snap/cmd_repair_repairs.go]: I like your suggestion a lot @degville !

**Pull request merged**